packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.22.0`
+
+*   Added: Unit tests for the MVS and JES functionality [#1632](https://github.com/zowe/vscode-extension-for-zowe/pull/1632).
+*   Added: Zowe Explorer is compatible with only Theia 1.18.0 or higher [#1643](https://github.com/zowe/vscode-extension-for-zowe/pull/1643).
```

Comment:
This is an ongoing item vs new item.  I don't think it will need to be in the CHANGELOG but can keep the note in the Zowe Explorer README